conversion/nhdr_write.py

```diff
@@ -121,7 +115,7 @@ def nhdr_write(nifti, bval, bvec, nhdr):
         if bval and bvec:
 
             mf = find_mf(spc_dir)
-            print(f'measurement frame: {matrix_string(mf)}')
+            print(f'measurement frame: {matrix_string(mf.T)}')
```

Comment:
This is done for consistency with `spc_dir` writing only:
`print(f'space directions: {matrix_string(spc_dir.T)} none')`